src/tests/ftest/util/test_utils_pool.py

```diff
@@ -1422,3 +1424,49 @@ def measure_rebuild_time(self, operation, interval=1):
         self.wait_for_rebuild_to_end(interval=interval)
         duration = time() - start
         self.log.info("%s duration: %.1f sec", operation, duration)
+
+    def check_pool_files(self, log, hosts, uuid, scm_mount):
+        """Check if pool files exist on the specified list of hosts.
+
+        Args:
+            log (logging): logging object used to display messages
+            hosts (NodeSet): list of hosts
+            uuid (str): uuid file name to look for in /mnt/daos.
+            scm_mount (str): SCM mount point such as "/mnt/daos". From test, it can be
+                obtained as self.server_managers[0].get_config_value("scm_mount")
```

Comment:
Does this function actually need to accept `uuid` or could that just come from `self.uuid.lower()`?
Also, I think `log` can just be `self.log` now, right?

Reply:
I checked the usage of `check_pool_files()` and noticed that `test_destroy_invalid_uuid()` in destroy.py updates the pool's `uuid` member to invalid one, then passes in valid UUID to this method, so we need this parameter.

I removed `log` parameter and just use `self.log`

Reply:
I see. I have an idea how to rework that but for now I think it's out of scope for this PR